docs/antrea-network-policy.md

```diff
@@ -253,7 +253,6 @@ spec:
         - protocol: TCP
           port: 5978
       name: DropToThirdParty
-      enableLogging: true
```

Comment:
I'm not sure why we removed these lines in sample yamls? Was it because we planned to deprecate it? Since it is no longer the case, it might be beneficial to leave some rules with `enableLogging: true` across sample yamls, so that when users simply copy-paste these they benefit from better debuggability

Reply:
Discussed offline, removing to avoid users encountering meter limit for other feature, and we have a separate example for `enableLogging` and `logLabel`.